src/OrchardCore.Themes/TheTheme/Recipes/themeSettings.recipe.json

```diff
@@ -0,0 +1,109 @@
+{
+  "name": "ThemeSettings",
+  "displayName": "Theme settings",
+  "description": "Enables the ThemeSettings content type.",
+  "author": "The Orchard Core Team",
+  "website": "https://orchardcore.net",
+  "version": "1.0.0-rc2",
+  "issetuprecipe": false,
+  "categories": [ "default" ],
+  "tags": [ "theme", "settings" ],
+  "steps": [
+    {
+      "name": "feature",
+      "disable": [],
+      "enable": [
+        "OrchardCore.CustomSettings"
+      ]
+    },
+    {
+      "name": "ContentDefinition",
+      "ContentTypes": [
+        {
+          "Name": "ThemeSettings",
+          "DisplayName": "Theme",
```

Comment:
I would name this `TheThemeSettings`. Then it would be unique to `TheTheme`, and can contain the reference to `TheTheme/favicon.ico` and then when someone changes themes, the `TheBlogThemeSettings` would activate and apply